test/Mono.Linker.Tests.Cases/Inheritance.Interfaces/InterfaceVariants.cs

```diff
@@ -20,29 +20,76 @@ public static void Main ()
 			t = typeof (UninstantiatedPublicClassWithImplicitlyImplementedInterface);
 			t = typeof (UninstantiatedPublicClassWithPrivateInterface);
 
-			UninstantiatedPublicClassWithInterface.InternalStaticInterfaceMethodUsed ();
-			InstantiatedClassWithInterfaces.InternalStaticInterfaceMethodUsed ();
-
+			UninstantiatedPublicClassWithInterface.InternalStaticInterfaceMethodUsedThroughImplementation ();
```

Comment:
What's the significance of `Internal` here? It's repeated a lot throughout the code. I can't think of a reason why that would make a difference.